### ISSUE TITLE
Spurious 99% inode usage warning reports  [sf#3552814]

### ISSUE THREAD

**Issue by monitoring-user** [monitoring-plugins/monitoring-plugins#1069]
Submitted by calestyo on 2012-08-01 00:39:01

Hi.

This is from the old / soon to be disabled again Nagios Plugins bug tracker that used to be at Nagios itself.
I've just copied this bug over. I'm not the original reporter and have no idea about the thoughts about this bug.
## bryanhunt:

Nagios is reporting that 99% of inodes on /home are used, when in fact only 1% are in use.

Subject: *\* PROBLEM Service Alert: localhost/Disk Space is WARNING **
Sent: 27 Jul 2012 17:41

****\* Nagios *****

Notification Type: PROBLEM
Service: Disk Space
Host: localhost
Address: 127.0.0.1
State: WARNING

Date/Time: Fri Jul 27 17:41:34 BST 2012

Additional Info:

DISK WARNING - free space: /home 4946 MB (20% inode=99%):

% sudo -u nagios /usr/lib/nagios/plugins/check_disk -m /home
DISK OK - free space: /home 17275 MB (72% inode=99%);| /home=6643MB;;;0;25198

% mount
/dev/mapper/dataloader-home on /home type ext3 (rw,relatime,errors=continue,data=ordered)

% df -ih /home
Filesystem Inodes IUsed IFree IUse% Mounted on
/dev/mapper/dataloader-home 1.6M 2.4K 1.6M 1% /home

% df -ih /
Filesystem Inodes IUsed IFree IUse% Mounted on
/dev/mapper/dataloader-root 1.3M 25K 1.3M 2% /

% df /home
Filesystem 1K-blocks Used Available Use% Mounted on
/dev/mapper/dataloader-home 25803068 6700352 17792188 28% /home 

System is AMD64, loads of cores.

I've had a try at building plugins from source but can't get it to build on that box. I've attached the output of trying to build nagios-plugins git head. 

---
## brennan:

The check is functioning correctly. The line that starts with "free space" in the check result is the key here...you have 99% of your inodes free. 

---
## bryanhunt:

Plugin works correctly.

% /usr/lib/nagios/plugins/check_disk -w 10% -c 20% -W 20% -K 10% -e

The problem was the plugin configuration, which was incorrect.

There might have been some copy-paste configuration going on. 

---
## bryanhunt:

Basically, what was happening, was that the command was being run like so:

/usr/lib/nagios/plugins/check_disk -w 90% -c 20% -W -e
DISK WARNING - free space: / 4402 MB (93% inode=98%); /dev 8035 MB (100% inode=99%); /run 1607 MB (99% inode=99%); /run/lock 5 MB (100% inode=99%); /tmp 9396 MB (99% inode=99%); /run/shm 3215 MB (100% inode=99%); /boot 198 MB (91% inode=99%); /home 17076 MB (71% inode=99%); /usr 6256 MB (78% inode=88%); /var 6718 MB (35% inode=98%);| /=302MB;495;3968;0;4960 /dev=0MB;803;6428;0;8035 /run=0MB;160;1285;0;1607 /run/lock=0MB;0;4;0;5 /tmp=11MB;991;7935;0;9919 /run/shm=0MB;321;2572;0;3215 /boot=17MB;22;181;0;227 /home=6841MB;2519;20158;0;25198 /usr=1759MB;844;6756;0;8445 /var=12429MB;2016;16133;0;20167

Note the missing parameter to -W

**Comment by RincewindsHat** [comment]
Looks like a "You're holding it wrong". Close the issue?
